libethereum/BlockQueue.h

```diff
@@ -67,6 +67,117 @@ enum class QueueStatus
 
 std::ostream& operator<< (std::ostream& os, QueueStatus const& obj);
 
+template<class T>
+class SizedBlockQueue
+{
+public:
+	std::size_t count() const
```

Comment:
Would it be better readable to put short one-liners on a single line?
The argument would be that the class then easily fits a single page and its behaviour can more easily be grasped.